app/resources/views/pages/profile/profile.blade.php

```diff
@@ -1,21 +1,20 @@
 @extends('layouts.default', ['body_class' => 'nav-profile'])
 @section('head-content')
-  <link href="/css/jquery/jquery-ui.css" rel="stylesheet" type="text/css">
-  <script src="/css/jquery/external/jquery/jquery.js"></script>
-  <script src="/css/jquery/jquery-ui.js"></script>
-  <script src="/js/profile.js"></script>
-  <script src="/js/profile_save_button.js"></script>
-  <script src="/js/profile_save_prompt.js"></script>
-  <script src="/js/utils.js"></script>
-  <script src="/js/question_explanation.js"></script>
-@stop
+
+	<link href="/css/jquery/jquery-ui.css" rel="stylesheet" type="text/css">
+	<script src="/css/jquery/external/jquery/jquery.js"></script>
+	<script src="/css/jquery/jquery-ui.js"></script>
+	<script src="/js/profile.js"></script>
+	<script src="/js/profile_save_button.js"></script>
+	<script src="/js/profile_save_prompt.js"></script>
+	<script src="/js/utils.js"></script>
+	<script src="/js/question_explanation.js"></script>
+@sto
```

Comment:
Fix the @stop missing the 'p' and remove these indentation changes.  The indentation changes here make it difficult to annotate who added each library and why because 'git blame' will show the same commit for all of these lines.